src/schrodinger/rdkit_extensions/stereochemistry.cpp

```diff
@@ -138,6 +138,28 @@ void wedgeMolBonds(RDKit::ROMol& mol, const RDKit::Conformer* conf)
         }
     }
 
+    // Save wiggly bonds (unknown stereochemistry) before
+    // ClearSingleBondDirFlags clears them. We check both the properties (set by
+    // MDL input) and BondDir (set when CXSMILES wiggly bonds are copied in
+    // mol_model.cpp).
+    std::vector<bool> had_wiggly_bond;
+    had_wiggly_bond.reserve(mol.getNumBonds());
+    for (auto bond : mol.bonds()) {
+        int wiggly_bond_v2000{0};
+        int wiggly_bond_v3000{0};
+        bond->getPropIfPresent(RDKit::common_properties::_MolFileBondStereo,
+                               wiggly_bond_v2000);
+        bond->getPropIfPresent(RDKit::common_properties::_MolFileBondCfg,
+                               wiggly_bond_v3000);
+        // MDL V2000 uses _MolFileBondStereo=4 for wiggly bonds
+        // MDL V3000 uses _MolFileBondCfg=2 for wiggly bonds
+        // (These values are from the MDL molfile specification, not defined as
+        // constants in RDKit)
+        bool is_wiggly = (wiggly_bond_v2000 == 4 || wiggly_bond_v3000 == 2 ||
+                          bond->getBondDir() == RDKit::Bond::BondDir::UNKNOWN);
+        had_wiggly_bond.push_back(is_wiggly);
+    }
+
```

Comment:
I would not do it this way: `WedgeMolBonds()` uses a heuristic to decide which bonds to wedge/dash. Even if it disfavors bonds between 2 chiral atoms, there are still chances that  one of the bonds you pick as wiggly ends up being chosen as a wedge/dash bond for a neighboring chiral atom, which would be overwritten with the wiggly bond when you restore them.

What I would do is store the chiral atom referred to by the wiggly bond, then turn the wiggly into, e.g. a wedge bond (a dash one would work too), and then, once  `WedgeMolBonds()` has done its thing, find out which of the bonds around the chiral atom has been wedged/dashed, and then replace that one with a wiggly.

Of course, all that also taking into account the properties.